package.json

```diff
@@ -8,7 +8,8 @@
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "whatwg-fetch": "^3.2.0"
```

Comment:
Inconsistent version ranges - strict for `react-scripts`, comparable with for others